x-pack/plugins/monitoring/public/components/summary_status/summary_status.scss

```diff
@@ -1,13 +1,10 @@
 .monSummaryStatusNoWrap {
   margin-left: $euiSizeM;
   margin-right: $euiSizeM;
+
   .euiTitle {
```

Comment:
@cchaos So that `EuiTitle` was part of an `EuiStat`. I adjusted the size using `titleSize`. I don't think truncation was working. I added a max-width to the flex items to get it working. I also added a class to the `EuiStat` and I'm using `eui-textTruncate` on the `p` tag inside it. Not 100% this is the way to go since I guess this could break if `EuiStat` were to ever replace that `p` tag with something else. Let me know if there's anything else you'd do.

Reply:
![image](https://user-images.githubusercontent.com/4016496/84427580-ff6fee80-abd9-11ea-9e05-7faace41dfa6.png)

Reply:
Thanks for in-depth analysis and the screenshot. I agree that targeting a generic HTML element rendered by EUI is not ideal, though it is slightly better than the `.eui` class.

Looking at the screenshot, I think there's probably a more ideal way to handle what happens if the window sizes are skinny, but that's out of scope for this particular initiative, so I think this is good for now. 👍